packages/ubuntu_bootstrap/test/installer_test.dart

```diff
@@ -126,11 +131,18 @@ extension on WidgetTester {
     when(locale.getLocale()).thenAnswer((_) async => 'en_US.UTF-8');
 
     final subiquityClient = MockSubiquityClient();
-    when(subiquityClient.getStatus()).thenAnswer(
-      (_) async => status.copyWith(state: ApplicationState.RUNNING),
-    );
-
-    registerMockService<ConfigService>(MockConfigService());
+    when(subiquityClient.getStatus()).thenAnswer((_) async => status);
+    when(subiquityClient.monitorStatus()).thenAnswer((_) =>
+        Stream.fromIterable([
+          status,
+          if (proceedToDone) fakeApplicationStatus(ApplicationState.DONE)
```

Comment:
This advances the installation state so that the loading indicator goes away and `pumpAndSettle()` can finish.